LICENSE.txt

```diff
@@ -0,0 +1,28 @@
+This software is OSI Certified Open Source Software.
+OSI Certified is a certification mark of the Open Source Initiative.
+
+Copyright (c) 2020, Enthought, Inc.
```

Comment:
Or 2018-2020 like the rest of the files in this project? Though I guess that range is copied from traits-future, which might be incorrect.

Reply:
sorry. i didn't notice the copyright in the rest of the modules. I think it should be 2020 everywhere.

Reply:
Cool. I will fix those in a separate PR then.